appveyor.yml

```diff
@@ -0,0 +1,16 @@
+environment:
+  nodejs_version: "6"
```

Comment:
Should we test multiple platforms too?
https://www.appveyor.com/docs/lang/nodejs-iojs/#testing-under-multiple-versions-of-nodejs-or-iojs